libtelemetry/src/full/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -407,12 +421,19 @@ private static synchronized void setAccessToken(@NonNull Context context, @NonNu
     if (TelemetryUtils.isEmpty(accessToken)) {
       return;
     }
-    if (sAccessToken.getAndSet(accessToken).isEmpty()) {
+    if (MapboxTelemetry.accessToken.getAndSet(accessToken).isEmpty()) {
       LocalBroadcastManager.getInstance(context)
         .sendBroadcast(new Intent(MapboxTelemetryConstants.ACTION_TOKEN_CHANGED));
     }
   }
 
+  @VisibleForTesting
+  static synchronized void clearAccessToken(@NonNull Context context) {
```

Comment:
if this is meant to be used only for tests we could `@RestrictTo(RestrictTo.Scope.TESTS)`